server/internal/storage/gorm/client.go

```diff
@@ -167,10 +166,7 @@ func (c *Client) Put(ctx context.Context, k *Key, v interface{}) (*Key, error) {
 		// Update all fields from model: https://gorm.io/docs/update.html#Update-Selected-Fields
 		rowsAffected := tx.Model(v).Select("*").Where("key = ?", k.Name).Updates(v).RowsAffected
 		if rowsAffected == 0 {
-			err := tx.Create(v).Error
-			if err != nil {
-				log.Printf("CREATE ERROR %s", err.Error())
-			}
+			tx.Create(v)
```

Comment:
Is ignoring errors ok here because this will just fail further on?

Reply:
Functionally speaking we always have ignored these errors. You can see that this method has an error return value, but that error is always `nil` whether or not the operation succeeds. This PR simply removes the log for consistency because we don't log any other database errors.

Addressing how we _handle_ database errors seems like it's outside the scope of this PR. I would like to see that happen before we add logging to this package because it may affect whether we need to log errors in this package or let them get caught by the logger at a higher level (e.g. the calling function or the server log interceptor). I created #322 to handle this.

Reply:
SGTM